services/worker/src/worker/job_runners/split/is_valid.py

```diff
@@ -48,19 +49,20 @@ def compute_is_valid_response(dataset: str, config: str, split: str) -> IsValidR
         kinds=SPLIT_HAS_PREVIEW_KINDS,
     )
 
-    try:
+    filter = False
+    search = False
+    with contextlib.suppress(Exception):
```

Comment:
(nit) in terms of coding style we use try/except instead of suppress, but not important

Reply:
ah ok. I'll revert

Reply:
indeed, the code is easier to read after that change